redisson/src/main/java/org/redisson/api/RAtomicDouble.java

```diff
@@ -106,6 +106,26 @@ public interface RAtomicDouble extends RExpirable, RAtomicDoubleAsync {
      * @param newValue the new value
      */
     void set(double newValue);
+    
+    /**
+     * Atomically sets the given value if current value is less than
+     * the special value
+     *
+     * @param less  compare value
+     * @param value newValue
+     * @return true when the value update is successful
+     */
+    boolean setIfLessThan(double less, double value);
```

Comment:
Sorry, I have checked and "than" is not used in similar method names. Could you please remove it?

Reply:
You mean change it to `setIfGreater` and `setIfLess` ？